ui/tests/acceptance/secrets/backend/kv/secret-test.js

```diff
@@ -20,6 +20,7 @@ import { runCmd } from 'vault/tests/helpers/commands';
 import { PAGE } from 'vault/tests/helpers/kv/kv-selectors';
 import codemirror from 'vault/tests/helpers/codemirror';
 import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { SECRET_ENGINE_SELECTORS as SS } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
```

Comment:
NIT: I've been calling these SES instead of SS.  
![image](https://github.com/user-attachments/assets/bd3da816-f13a-4f4f-a644-23fea732ed21)